pennylane/devices/__init__.py

```diff
@@ -84,6 +84,31 @@
     transforms.split_non_commuting
     transforms.hamiltonian_expand
 
+
+Modifiers
+---------
+
+The ``modifiers`` allow for the easy addition of default behavior to a device.
+
+.. currentmodule:: pennylane.devices.modifiers
+.. autosummary::
+    :toctree: api
+
+    convert_single_circuit_to_batch
+    simulator_tracking
+
+For example with a custom device we can add simulator-style tracking and the ability to handle a single circuit.
```

Comment:
I see you give more details in each transform, but I will admit I saw this example and wanted to be certain of what they actually do without considering looking deeper... maybe it's just me, but can you just add a message after the code snippet saying to view each modifier for more details?

Reply:
Add an example of using the device too.